articles/Bash_vs_PowerShell.md

```diff
@@ -0,0 +1,20 @@
+
+# Differences between Bash and Powershell
+
+## A shell is a command line interface that enables the user to interact with the computer. It is named a shell because it is the outermost layer around the operating system.
+
+| Bash (Bourne-Again shell) | Microsoft PowerShell|
+| ----------- | ----------- |
+| Bash is a Linux based command line interface (CLI) used for communication or interaction between the operating system and end user.   | PowerShell was first developed for Windows systems, but now also available for Unix-like platforms. PowerShell Core is a cross-platform (Windows, Linux, and macOS) command-line shell, and interactive scripting language. |
+| Bash Shell is written by Brian Fox and developed by the GNU project. It was first released in 1989.  | Jeffrey Snover is the Inventor and Architect of Windows PowerShell. It was introduced in 2006.       |
+| It is mainly prepared for Linux and Unix operating system from the first day   | It can execute on any version of Windows from Window 97 to Windows 10.      |
+| Bash treats outputs as strings  | PowerShell treats output as objects.        |
+| It passes output and input as plain text, which means it is easy for the user to move information to the next program.  | PowerShell scripts share complex data, passing entire data structures between commands. PowerShell is object-oriented scripting language.|
+| The Bash shell and command language don't offer these capabilities in Windows.   | PowerShell is designed especially for the System administrators to enables them to perform a task on remote and local windows system through full access to COM and WMI. It allows administrators to manage both the workloads windows server and Linux server where production application are hosted.       |
```

Comment:
Perhaps, a line or two about `COM` and `WMI` could be added

Reply:
Okay!